propka/lib.py

```diff
@@ -198,6 +198,8 @@ def build_parser(parser=None):
        Argument `--generate-propka-input` has been removed as writing PROPKA
        input files is no longer supported.
     """
+    import propka
```

Comment:
Does this create a cyclical import?

And even if the cyclical import is delayed (because the import happens inside a function, not the top level), would there be a different way to get this information?

Reply:
Additional thought:  use directly `import ._version` as created by versioneer?

Reply:
It will not create a circular import, even if moved out of the function to the top level. `propka.__version__` is already assigned at that point so I don't see a reason to get the information a different way.